src/opcodes/opcodes.cairo

```diff
@@ -2,6 +2,7 @@ pub mod Opcode {
     pub const OP_0: u8 = 0;
     pub const OP_1: u8 = 81;
     pub const OP_ADD: u8 = 147;
+    pub const OP_DEPTH: u8 = 116;
```

Comment:
Please move this before OP_ADD so things will be in order, which will make things easier to maintain and see later on

Reply:
@b-j-roberts done